test/component/deceased/testAddressNew.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
```

Comment:
Could you rename this file to testAddress_new_death_cert_flow.js? Just to match testMultipleApplications_new_death_cert_flow.js